eoscompanion/main.py

```diff
@@ -157,6 +157,31 @@ def do_activate(self):  # pylint: disable=arguments-differ
         logging.info('Activated')
         return Gio.Application.do_activate(self)
 
+CONFIG_FILE = '/etc/eos-companion-app/config.ini'
+CONFIG_SECTION = 'Companion App'
+CONFIG_NAME = 'loglevel'
+DEFAULT_LOG_LEVEL = logging.INFO
+
+def getLogLevel():
```

Comment:
Function naming style should be PEP8 complaint here (`get_log_level`). Also, please add a docstring.

Reply:
Fixed